src/bundle/Resources/public/js/scripts/button.state.toggle.js

```diff
@@ -1,21 +1,27 @@
-(function(global, doc) {
```

Comment:
Nice! 👍 

More for @dew326 to give input on regarding coding style, but technically this could also be something ala `document.addEventListener('DOMContentLoaded', function() {` now. Not that it would be important, but it would avoid the indentation change for the additional nesting, but on the other hand it will need to refer to document insted of doc inside fucntion.